README.md

```diff
@@ -338,6 +338,10 @@ environment.rb for example):
 Jbuilder.deep_format_keys true
 ```
 
+## Testing JBuilder Response body with RSpec
+
+To test the response body of your controller spec, enable `render_views` in your RSpec context. This [configuration](https://relishapp.com/rspec/rspec-rails/v/5-1/docs/controller-specs/render-views) renders the views in a controller test.
```

Comment:
```suggestion
To test the response body of your controller spec, enable `render_views` in your RSpec context. This [configuration](https://rspec.info/features/6-0/rspec-rails/controller-specs/render-views) renders the views in a controller test.
```

Rspec docs are no longer hosted on relishapp.com, suggestion fixes link to working docs

Reply:
Thank you @K4sku :tada: